docs/resources/aws_rds_group_option.md

```diff
@@ -0,0 +1,76 @@
+---
+title: About aws_rds_group_option Resource
+---
+
+# aws\_rds\_option\_group
+
+Use the `aws_rds_group_option` InSpec audit resource to test detailed properties of an individual RDS cluster.
+
+RDS gives you access to the capabilities of a MySQL, MariaDB, PostgreSQL, Microsoft SQL Server, Oracle, or Amazon Aurora database server.
+
+## Syntax
+
+An `aws_rds_group_option` resource block uses resource parameters to search for an RDS option group, and then tests that RDS option group.  If no RDS option group match, no error is raised, but the `exists` matcher will return `false` and all properties will be `nil`. 
+    describe aws_rds_group_option(engine_name: 'mysql') do
+      it { should exist }
+    end
+
+    # Can also use hash syntax
+    describe aws_rds_group_option(option_group_name: 'default:aurora-5-6') do
+      it { should exist }
+    end
+
+#### Parameters
+
+##### option\_group\_name or engine\_name _(required)_
+
+This resource accepts a single parameter, the user-supplied cluster identifier. This parameter isn't case-sensitive.
+This can be passed either as a string or as a `option_group_name: 'value'` key-value entry in a hash.
+
+See also the [AWS documentation on RDS cluster](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/Aurora.Overview.html).
```

Comment:
Change the link

Reply:
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-rds-optiongroup.html